core/PRP/Geometry/plDrawableSpans.cpp

```diff
@@ -744,54 +742,29 @@ void plDrawableSpans::setSpaceTree(plSpaceTree* tree)
     fSpaceTree = tree;
 }
 
+size_t plDrawableSpans::IFindBufferGroup(unsigned int format, unsigned int vertsNeeded)
+{
+    // For some reason, Plasma attempts to maintain only one vertex storage buffer per group
+    for (size_t i = 0; i < fGroups.size(); ++i) {
+        plGBufferGroup* group = fGroups[i];
+        if (group->getFormat() == format)
+            if (group->getNumVertices() + vertsNeeded < plGBufferGroup::kMaxNumVertsPerBuffer)
+                return i;
```

Comment:
Please use braces around multiline if bodies, even if they are not syntactically necessary.